pyk/src/pyk/ktool/kfuzz.py

```diff
@@ -20,13 +20,32 @@
     from ..kore.syntax import Pattern
 
 
+class KFuzzTestPrinter:
+    def print_test_prologue(self) -> None:
+        pass
```

Comment:
Consider

```suggestion
    def print_test_prologue(self) -> str:
        pass
```

for all methods.

Reply:
I actually would prefer this stays `None` and allow more flexibility. Primarily this would be used to report the status of the fuzzing to the user, but as far as how that's done, either with `print` or `logging` or some other creative way should be up to the application.

Given that, I've renamed it from `KFuzzTestPrinter` to `KFuzzTestCaseHandler`. That's a bit of a long name, open to suggestions.